### ISSUE TITLE
ModuleNotFoundError: No module named 'packaging'

### ISSUE THREAD

**Issue by arvoelke** [nengo/nengo-dl#158]
It looks like since commit bf887742d06e07e429e4610b9ad842af097c07c1 in PR #149, I am getting 
```
  File "/home/arvoelke/git/nengo-dl/nengo_dl/__init__.py", line 49, in <module>
    from nengo_dl import (
  File "/home/arvoelke/git/nengo-dl/nengo_dl/op_builders.py", line 23, in <module>
    from nengo_dl import utils
  File "/home/arvoelke/git/nengo-dl/nengo_dl/utils.py", line 14, in <module>
    from packaging import version
ModuleNotFoundError: No module named 'packaging'
```
on import. I'm installing the current master branch (`pip install -e .`) which doesn't appear to pick up the `packaging` library. Looking at [a recent build](https://travis-ci.org/github/nengo/nengo-dl/jobs/684780224), it seems to be getting picked upstream by one of the test dependencies?

**Comment by drasmuss** [comment]
Hmm it definitely used to be pulled in on any installation, but maybe something changed in the upstream dependencies so that is no longer the case. Will just add it as a direct dependency in any case.

**Comment by hunse** [comment]
Yeah, I think it's typically included with one of setuptools/distutils/pip, but I think I ran into one machine once where it wasn't installed.

**Comment by drasmuss** [comment]
This was fixed in #159
